docs/modules/ROOT/partials/install/prepare-syn-config-terraform.adoc

```diff
@@ -49,15 +49,6 @@ yq eval -i ".parameters.openshift4_terraform.terraform_variables.rhcos_template
   ${CLUSTER_ID}.yml
 ----
 
-endif::[]
-ifeval::["{provider}" == "cloudscale"]
-. Configure cloudscale.ch-specific Terraform variables
-+
-[source,bash,subs="attributes+"]
-----
-yq eval -i ".parameters.openshift4_terraform.terraform_variables.image_slug = \"custom:rhcos-{ocp-minor-version}\"" \
-  ${CLUSTER_ID}.yml
```

Comment:
We now set this variable in https://git.vshn.net/syn/commodore-defaults/-/blob/5de93dc03471c160e3887197db140745584ab93b/distribution/openshift4/cloud/cloudscale/machinesets.yml#L56 (internal)